tests/unit/test_filters.py

```diff
@@ -16,8 +16,8 @@ def set_up_for_filters():
         section=section,
     )
 
-    theme1 = factories.ThemeFactory(keywords=["dog", "puppy"], question=question)
-    theme2 = factories.ThemeFactory(keywords=["cat", "kitten"], question=question)
+    theme1 = factories.ThemeFactory(theme_number=1, keywords=["dog", "puppy"], question=question)
```

Comment:
I think we can get FactoryBoy to auto-increment these: https://factoryboy.readthedocs.io/en/stable/reference.html#factory.Sequence